### PR TITLE
Updating additional dependencies for landrush (Fedora -> Fedora 23/24)

### PR DESCRIPTION
**Pull request by amitkrout**
…/24)

### CODE REVIEW

**Comment by brgnepal** [comment]
LGTM

File: docs/installing.rst [hunk 1]
```diff
@@ -154,7 +154,7 @@ The following virtualization providers are suggested for the respective operatin
 
 For some operating systems, you might need to install additional dependencies before you install the Vagrant plugins.
 
-* Fedora 24
+* Fedora 22/23/24
```
Comment (praveenkumar) [inline docs/installing.rst]: Don't put F22 it's already end of life and we are not going to support it.

Comment (brgnepal) [inline docs/installing.rst]: @praveenkumar It's there in other places too. So I would probably let it be here and clean it later.

Comment (LalatenduMohanty) [inline docs/installing.rst]: I think we should take this opportunity to clean up all F22 entries in the documentation as F22 is already end of life.

**Comment by LalatenduMohanty** [comment]
Merging this as it is. We can remove F22 in another PR.

**Comment by LalatenduMohanty** [comment]
@amitkrout are you planning to fix the PR as per comments?

**Comment by amitkrout** [comment]
@LalatenduMohanty Updated the PR as per the comments.

**Comment by LalatenduMohanty** [comment]
LGTM

**Pull request merged**